### PR TITLE
new event syntax

### PR DESCRIPTION
**Pull request by jordansread**
for #174 

@jiwalker-usgs can you take a look?

### CODE REVIEW

File: target/fishGDDFig-desktop.svg [hunk 1]
```diff
@@ -214,15 +214,12 @@
...
-								i++
-							} else {
-			                   clearInterval(delaySendValue);
-							}}, 2000);
-			   
+			var toSend = getPtValue(0);
+			for (var i = 1; i < dots.length; i++) {
+				toSend = toSend + "," + getPtValue(i);
+			}
+			toSend = "{'walleye':["+toSend+"]}"
```
Comment (jiwalker-usgs) [inline target/fishGDDFig-desktop.svg]: I'm using `JSON.stringify()`, which might make this better.

File: target/fishGDDFig-desktop.svg [hunk 2]
```diff
@@ -347,15 +344,15 @@
             
 			if (canFinish()){
     			svgDoc.getElementById("im-done").setAttribute('class','enabled');
-    			svgDoc.getElementById("im-done").setAttribute('onclick', 'imDone();window.parent.ga("send", "event", "svg interaction", "click", "guessed GDD");')
+    			svgDoc.getElementById("im-done").setAttribute('onclick', 'imDone();window.parent.ga("send", "event", "figure", "submitted GDD guess");')
```
Comment (jiwalker-usgs) [inline target/fishGDDFig-desktop.svg]: might just want to put the ga code in the imDone() function

File: target/fishGDDFig-desktop.svg (hunk 1 above)
Comment (jordansread) [reply target/fishGDDFig-desktop.svg]: so I would use `window.parent.JSON.stringify()`?

Comment (jiwalker-usgs) [reply target/fishGDDFig-desktop.svg]: Do you have access to JSON from there? It is a non-scoped function, so should just work.

File: target/fishGDDFig-desktop.svg (hunk 2 above)
Comment (jordansread) [reply target/fishGDDFig-desktop.svg]: yeah, will change that

**Comment by jordansread** [comment]
@jiwalker-usgs I think I get 'em

**Pull request merged**